docs/05-technical-reference/00-architecture/evnt-01-architecture.md

```diff
@@ -26,11 +26,11 @@ This diagram explains the event flow in Kyma, from the moment the Application se
 
 ## Event Publisher Proxy
 
-Event Publisher Proxy receives legacy and Cloud Event publishing requests from the cluster workloads (microservices or Functions). It converts any legacy events to Cloud Events. Then, it redirects events to the NATS server. It also fetches a list of Subscriptions for a connected Application.
+Event Publisher Proxy receives legacy and Cloud Events and publishes them to the configured eventing backend. All the legacy events are automatically converted to Cloud Events.
 
 ## Eventing Controller
 
-Eventing Controller manages the internal infrastructure in order to receive an event. It watches Subscription Custom Resource Definitions. When an event is received in an Application, it lays down the Eventing infrastructure in NATS in order to trigger a Function. Eventing Controller also dispatches messages to subscribers such as a Function or another workload.
+Eventing Controller manages the internal infrastructure in order to receive an event. It watches Subscription Custom Resources. When an event is received, it dispatches the message to the configured sink.
```

Comment:
```suggestion
Eventing Controller manages the internal infrastructure in order to receive an event. It watches Subscription Custom Resources. When an event is received, Eventing Controller dispatches the message to the configured sink.
```
"it" was ambiguous here (referring to "an event", or the Eventing Controller?) so I took a guess - correct me if I guessed wrong :D